www/pages/stop/stop-controller.js

```diff
@@ -108,7 +110,9 @@ angular.module('pvta.controllers').controller('StopController', function ($scope
               var times = {sExact: moment(departure.SDT).format('LT'),
                            eExact: moment(departure.EDT).format('LT'),
                            sRelative: moment(departure.SDT).fromNow(),
-                           eRelative: moment(departure.EDT).fromNow()};
+                           eRelative: moment(departure.EDT).fromNow(),
+                           eRelativeNoPrefix: moment(departure.EDT).fromNow(true)
```

Comment:
By default `fromNow` returns `"in XX minutes"`,  but we can have it without the `in` prefix, and I thought that looked better for when we display it on the unexpanded tile.